planner/core/find_best_task.go

```diff
@@ -1629,6 +1631,19 @@ func getMostCorrCol4Index(path *util.AccessPath, histColl *statistics.Table, thr
 	return nil, corr
 }
 
+// keepAccessCondsAsFilter4TablePlan is used to keep the access conditions as
+// filter conditions when the following conditions are met:
+// 1. Filter conditions are nil.
+// 2. Access conditions satisfies the conditions of the
+// MaybeOverOptimized4PlanCache function.
+func keepAccessCondsAsFilter4TablePlan(ctx sessionctx.Context, accessConds []expression.Expression, filterConds []expression.Expression) []expression.Expression {
```

Comment:
How about `keepAccessCondsAsFilter4PlanCache`?
For all particular functions for plan cache, we use `4PlanCache` as their suffix.